### DISCUSSION TITLE
[V5] Cron job

### DISCUSSION THREAD

**Discussion by luisnovaisdev** [adonisjs/core#2747] [Help] [answered]
Can I usa node-cron with adonisjs ? 
If I can, can you explain how is that possible ? 

If can't, how can I do cron jobs in adonisjs ?

**Comment by Vishwas-verma** [comment]
Yes uh can use node-cron with AdonisJS but I won't recommend using it , instead using [adonis-scheduler](https://www.npmjs.com/package/adonis-scheduler) will solve your most requirement.

**Reply by luisnovaisdev** [reply]
This package dont work in version 5 , just in the preview

**Comment by d4n1b** [comment, marked as answer]
Hi @luisnovaisdev ,

You can just create a file under `start` folder:

```ts
// start/cron.ts
import cron from 'node-cron';

cron.schedule('*/10 * * * * *', async () => {
  // Your code here  
});
```

And make sure you add it to `.adonisrc.json`

```json
{
  // ...
  "preloads": [
    "./start/cron", // <--
    "./start/routes",
    "./start/kernel"
  ]
}
```

This should be enough.
I hope it helps

**Reply by luisnovaisdev** [reply]
Thanks you 👌
